Srpg 77 tileset 메인 퀘스트 1-1 ~ 1-5 타일셋 배치 완료

jhwoo944 opened the pull request:
## 개요
<!---- 변경 사항 및 관련 이슈에 대해 간단하게 작성해주세요. 어떻게보다 무엇을 왜 수정했는지 설명해주세요. -->
 - 메인 퀘스트 1-1 부터 1-5까지의 타일맵 배치 작업을 완료했습니다. 
 0-Scenes / Test / TileTestScene / MainQuest에서 확인할 수 있습니다.
 칸 당 타일 크기는 16x16 pixel 입니다.
 (맵 구조에 대한 설명은 슬랙 - 파일링크 - 파밍 / 모험 스테이지 기획서 PPT를 참고해주세요.)

 - 타일셋 에셋과 팔레트를 모아놓은 폴더 TileSets을 추가했습니다.
 - 아래의 내용을 적용했습니다.
 1. projectSetting -> Graphics Trasparency Sort Axis의 Y값 1로 변경
 2. Tilemap Renderer Mode Mode Individual로 변경

 - 애니메이션 타일을 적용했으나 현재 Develop 브랜치에 오류가 있는 관계로 테스트해보지는 못했습니다.

Todo.
 - 파밍 스테이지 4종 타일셋 배치하기
 - 타일셋 배치한 맵을 토대로 전투 테스트 해보기
 - 적절한 적 / 맵 기믹 / 스테이지 클리어 조건 / 클리어 보상 / 스토리를 배치하고 실제 스테이지로 완성시키기

*참고용으로 각 맵 별 사진 첨부합니다.
 - 1-1
![image](https://github.com/NBC-SRPG/SRPG/assets/128718414/9056e632-76a8-4ba6-8492-01d2f636eadc)
![image](https://github.com/NBC-SRPG/SRPG/assets/128718414/9bb19f74-718f-46d4-9449-491328d81d5c)

 - 1-2
![image](https://github.com/NBC-SRPG/SRPG/assets/128718414/ce1649d3-6e12-43bb-8f2f-80fb4a357a5a)
![image](https://github.com/NBC-SRPG/SRPG/assets/128718414/6553b337-40a6-4999-aae3-0de8d9995895)

 - 1-3
![image](https://github.com/NBC-SRPG/SRPG/assets/128718414/a2b8d837-4fce-4138-887b-59e196dacaf7)
![image](https://github.com/NBC-SRPG/SRPG/assets/128718414/386cd69f-d5cd-442c-a77c-cb96c29a796a)

 - 1-4
![image](https://github.com/NBC-SRPG/SRPG/assets/128718414/969c33fa-aae3-4db1-ab91-791666d46916)
![image](https://github.com/NBC-SRPG/SRPG/assets/128718414/23960729-114b-44a5-aa93-b90c46dfa375)

 - 1-5
![image](https://github.com/NBC-SRPG/SRPG/assets/128718414/782eff87-35db-410b-8d91-005f13721e66)
![image](https://github.com/NBC-SRPG/SRPG/assets/128718414/9d193c32-66b0-44d4-9040-7d916596d1a8)


<!---- Resolves: #(Isuue Number) -->

## PR 유형
어떤 변경 사항이 있나요?

- [x] 새로운 기능 추가
- [ ] 버그 수정
- [ ] CSS 등 사용자 UI 디자인 변경
- [ ] 코드에 영향을 주지 않는 변경사항(오타 수정, 탭 사이즈 변경, 변수명 변경)
- [ ] 코드 리팩토링
- [ ] 주석 추가 및 수정
- [ ] 문서 수정
- [ ] 테스트 추가, 테스트 리팩토링
- [ ] 빌드 부분 혹은 패키지 매니저 수정
- [x] 파일 혹은 폴더명 수정
- [ ] 파일 혹은 폴더 삭제

## PR Checklist
PR이 다음 요구 사항을 충족하는지 확인하세요.

- [x] 커밋 메시지 컨벤션에 맞게 작성했습니다. (ex. feature/SRPG-11-UIBase)
- [ ] 변경 사항에 대한 테스트를 했습니다.(버그 수정/기능에 대한 테스트).

jhwoo944 committed:
SRPG-77 TileSet Slice

 - 타일셋 스프라이트가 유니티 에셋 용으로 제작된 것이 아니어서,
스프라이트 설정을 알맞게 조정하고
스프라이트 에디터를 통해 알맞은 픽셀 크기로 슬라이스했습니다.

jhwoo944 committed:
Merge branch 'develop' into SRPG-77-Tileset

jhwoo944 committed:
SRPG-77 Animation Tile

 - 시험용 애니메이션 타일 추가
 - 타일셋 에셋 경로 변경

jhwoo944 committed:
SRPG-77 Feat: TileSet Add

 - 타일셋 에셋 추가 및 슬라이스
 - 테스트 씬 작업중

jhwoo944 committed:
SRPG-77 Feat: Tileset Slice

 - 이미지가 너무 커서 유니티 내부에서 화질을 낮춰버리는 타일셋을 직접 알맞은 크기로 잘라서 타일셋에 적용 시켰습니다.
 - 또한, 스테이지 내용을 기획하고 씬을 만들었습니다.
 - 내일부터 타일 본격적으로 깔 예정입니다.

jhwoo944 committed:
SRPG-77 Feat: MainQuest 1-1, 1-2 TileSet

 - 메인퀘스트 1-1과 1-2씬의 타일셋 배치 작업을 끝냈습니다.
 - 현재 1-3 작업중이며, 주말 (4/6 ~ 4/7) 중으로
메인퀘스트 1-3, 1-4, 1-5 및 파밍 던전 4종까지 타일셋 배치 작업 완료할 예정입니다.

jhwoo944 committed:
SRPG-77 Feat: TileMap 1-3

 - 메인퀘스트 1-3 타일맵 작업 완료

jhwoo944 committed:
SRPG-77 Feat: 1-4

 - 1-4 타일맵 작업중

jhwoo944 committed:
Merge branch 'develop' into SRPG-77-Tileset

jhwoo944 committed:
SRPG-77 FeatL TileMap

 - 1-4 작업중
 - 1-3에 배경 타일 추가

jhwoo944 committed:
Merge branch 'develop' into SRPG-77-Tileset

jhwoo944 committed:
SRPG-77 Feat: TileSet

 - 메인퀘스트 1-5 타일셋 작업 완료
 - 1-1 ~ 1-4 미흡한 부분 보완

jhwoo944 committed:
SRPG-77 Fix: 오류 수정

 - 일부 짝이 안 맞춰진 타일 블록을 제대로 맞물리게 수정

jhwoo944 committed:
SRPG-77 Fix: 수정 보완

 - 일부 미흡한 타일 배치 수정 보완

jhwoo944 committed:
Merge branch 'develop' into SRPG-77-Tileset

jhwoo944 committed:
SRPG-77 Feat: CharacterExp Stage

 - 캐릭터 Exp 스테이지 타일맵 작업 완료

jhwoo944 committed:
SRPG-77 Feat: 골드 수급 스테이지 타일셋

 - 골드수급 스테이지 (지역 수비형) 타일셋 작업 진행 중 중간 커밋

SeRDic0705 reviewed (changes requested):
이전에 말했던대로, 타일셋폴더는 PullRequest에 포함하지 않도록하고, 맵은 프리팹으로Addressable에 올려주세요~
로딩이너무 오래걸려요 ㅠㅠ
![image](https://github.com/NBC-SRPG/SRPG/assets/49552752/905cdfd6-950a-4bb7-ae12-5e5f33b36dff)

jhwoo944 committed:
SRPG-77 Feat: 파밍스테이지 3종 타일맵 작업

 - 파밍스테이지(경험치/골드/조각) 맵 90%쯤 구현 완료
 - 남은 작업 : 파밍스테이지(보너스맵) + 파밍스테이지(장비성장재료)

jhwoo944 committed:
Merge branch 'develop' into SRPG-77-Tileset

jhwoo944 committed:
SRPG-77 Feat: Tileset Prefeb

 - 작업한 타일맵 프리펩으로 저장
 - 깃 이그노어에 /[As]sets/[Ti]leSets/ 추가
(타일 팔레트 및 타일 에셋 제외하도록 변경)

jhwoo944 committed:
-

-

jhwoo944 committed:
SRPG-77 Fix:

 - 타일셋 폴더 이름 변경

jhwoo944 committed:
-

-

jhwoo944 committed:
-

-

jhwoo944 commented:
타일셋 작업물은 Feature-3에 merge 되어 있습니다.
혹은 TileSet-ingoreTest 브런치를 참고하면 됩니다.
